### ISSUE TITLE
Error: Cannot find module 'babel/polyfill'

### ISSUE THREAD

**Issue by substack** [axross/tap-diff#5]
I just installed tap-diff and I get this error:

```
$ node test/del.js | tap-diff 
module.js:339
    throw err;
    ^

Error: Cannot find module 'babel/polyfill'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/home/substack/prefix/lib/node_modules/tap-diff/distributions/index.js:9:22)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: write EPIPE
    at Object.exports._errnoException (util.js:874:11)
    at exports._exceptionWithHostPort (util.js:897:20)
    at WriteWrap.afterWrite (net.js:763:14)
```

**Comment by mpj** [comment]
Same here

**Comment by axross** [comment]
This problem was solved by v0.1.1! Thank you for your issue!
